Yosemite/YosemiteTests/Stores/RefundStoreTests.swift

```diff
@@ -555,7 +555,8 @@ private extension RefundStoreTests {
             shippingLines: [],
             coupons: [],
             refunds: [],
-            fees: []
+            fees: [],
+            taxes: [] // TODO: 5809 - Add sampleOrderTaxLines method
```

Comment:
❓ I'm curious: why do we need `sampleOrderTaxLines` in `RefundStoreTests`?

Reply:
😀 My bad. Removed the TODO in https://github.com/woocommerce/woocommerce-ios/pull/5916/commits/21ea63c8fe86a860d9f7784404ecf69b2ecdb2cc